React/Views/RCTRefreshControl.m

```diff
@@ -52,18 +57,9 @@ - (void)beginRefreshing
 {
   // When using begin refreshing we need to adjust the ScrollView content offset manually.
   UIScrollView *scrollView = (UIScrollView *)self.superview;
-  CGPoint offset = {scrollView.contentOffset.x, scrollView.contentOffset.y - self.frame.size.height};
-
-  // `beginRefreshing` must be called after the animation is done. This is why it is impossible
-  // to use `setContentOffset` with `animated:YES`.
-  [UIView animateWithDuration:0.25
-                          delay:0
-                        options:UIViewAnimationOptionBeginFromCurrentState
-                     animations:^(void) {
-                       [scrollView setContentOffset:offset];
-                     } completion:^(__unused BOOL finished) {
-                       [super beginRefreshing];
-                     }];
+  CGPoint offset = {scrollView.contentOffset.x, scrollView.contentOffset.y - RCTRefreshControl.defaultHeight};
```

Comment:
Could you also add a small comment that explains why we can't use `self.frame.size.height` here.

Reply:
when we use self.frame.size.height, it has a issue when we touch and hold to pull at list but just scroll to a part of loading indicator show on screen and then we release it, the height not correct -> beginRefresh will wrong, indicator will not show in nexttime